code/style.css

```diff
@@ -5,3 +5,326 @@ this can be used for the FAQ-accordion. read more here: https://www.w3schools.co
 .active + .answer {
   /* some styling here */
 }
+body{
+  margin: 0px;
+  padding: 0px;
+  font-family: 'Montserrat', sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color:rgb(136, 177, 177);
+}
+header{
+  width: 100%;
+}
+img{
+  object-fit: cover;
+  width: 100%;
+  height: auto;
+  top: 0px;
+}
+
+h1{
+  font-size: 30px;
+  color: white;
+  z-index: 1;
+  text-align: center;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
I think you can adapt the font-family to the whole body and don't need to put again in below categories! Or maybe there is a reason why you have repeated this element? :)